### ISSUE TITLE
Text pane rendering regression on CLX-fb

### ISSUE THREAD

**Issue by froggey** [McCLIM/McCLIM#484]
Using the CLX-fb backend:
![clx-fb-inspector](https://user-images.githubusercontent.com/72612/40546430-2ad36170-6027-11e8-8340-c84e128d1778.png)
The CLX backend behaves as expected:
![clx-inspector](https://user-images.githubusercontent.com/72612/40546491-5690186c-6027-11e8-9038-68279b9b1c6b.png)

This occurs in the inspector, puzzle demo and beirc.
git bisect suggests that this was introduced in commit 941216300132c8e62208f929f8b6312801bcaa10

**Comment by nsrahmad** [comment]
Other panes do re-draw their backgrounds afterwards. But they used `medium-background` to get the background color to use. `medium-background` uses parent sheets background-color. which has no problem with default CLX backend where each pane is mirrored and has unique parents. You could see the same problem with CLX backend if you run it single-mirrored with:
```
(let ((clim:*default-server-path*  '(:clx :mirroring :single :font-renderer mcclim-truetype::truetype-font-renderer)))
    (clim-demo:demodemo))
```
Using `pane-background` as repaint backgound ink fixes the problem.

**Comment by lokedhs** [comment]
As a fix for this 769a87d47c3f4e92912e5c557d0b5b804b9f0232 was committed. However, this commit causes a problem where the background colour is no longer updated when shift-selecting text.

To reproduce, shift-drag over text in an application pane.

**Comment by nsrahmad** [comment]
@lokedhs  I knew about that. As I said above there seems to be a bug in `medium-backgound` for single mirrored sheets. For a temporary fix, using `pane-background` should work.
